filebeat/docs/reload-configuration.asciidoc

```diff
@@ -47,6 +62,50 @@ definitions. For example:
   scan_frequency: 5s
 ------------------------------------------------------------------------------
 
+
 WARNING: It is critical that two running prospectors DO NOT have overlapping
 file paths defined. If more than one prospector harvests the same file at the
 same time, it can lead to unexpected behaviour.
+
+[float]
+[[reload-module-config]]
+=== Module config
+
+For module configurations, you specify reload options in the
+`filebeat.config.modules` section of the +{beatname_lc}.yml+ file. For example:
+
+[source,yaml]
+------------------------------------------------------------------------------
+filebeat.config.modules:
+  enabled: true
+  path: configs/*.yml
+  reload.enabled: true
+  reload.period: 10s
+------------------------------------------------------------------------------
+
+TIP: If you are using the <<modules-command,`modules`>> command to enable and
```

Comment:
@exekias I'm not sure if I'm going far enough here. I think it might be better to show the modules.d path here. WDYT? (still need to write those docs...tomorrow)